### ISSUE TITLE
Tabs V2

### ISSUE THREAD

**Issue by saurabhsutar192** [antstackio/hover-design#66]
Enhancement to existing Tab Component.
#41 

Specs:
1. Accessibility - should be controllable by keyboard.
2. Orientation
3. Variants - Outline, Pills
4. Refs for individual Tab.

**Comment by pruthvi2103** [comment]
pl add the stuff we mentioned for context. For eg: add

**Comment by pruthvi2103** [comment]
closed by #75
